region/brinstar/kraid/Kraid Room.json

```diff
@@ -253,6 +253,40 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "X-Mode (Kraid Alive) - R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"not": "f_DefeatedKraid"},
+        "canRiskPermanentLossOfAccess",
+        {"or": [
+          "h_CrystalFlash",
+          {"and": [
+            {"refill": ["Energy"]},
```

Comment:
This needs DisableEtanks or canbeExtremely?Patient.  Or another stack of canRiskPermanentLossOfAccess to not collect etanks.

Reply:
Comparing to farming bugs:

Nail drop rate is 0% Nothing and slightly more than 3-to-1 biased towards Large Energy when full on ammo. In terms of drop rate it is superior to Zebs (2% Nothing and even split between small/large) and worse than Zebbo (0% Nothing and 0% Small Energy).

Farm cycle is notably worse than both: only one nail can be farmed (the other must be avoided or destroyed), and this must be done while evading Kraid and his spike missiles.

I'm thinking maybe somewhere between `canBePatient` and `canBeVeryPatient`? Definitely `canDodgeWhileShooting`, maybe `canTrickyDodgeEnemies`?